src/CMakeLists.txt

```diff
@@ -282,11 +281,13 @@ if(MSVC)
     target_link_options(woof-setup PRIVATE "/MANIFEST:NO")
 endif()
 
-add_custom_command(TARGET woof POST_BUILD
+add_custom_target(woof-base
     COMMAND ${CMAKE_COMMAND} -E tar cf
-    "$<TARGET_FILE_DIR:woof>/woof.pk3" --format=zip .
+    "${CMAKE_CURRENT_BINARY_DIR}/woof.pk3" --format=zip .
     WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}/base")
 
+add_dependencies(woof woof-base)
```

Comment:
This *always* rebuilds the PK3 file:
```
$ git branch | grep ^\*
* id24_sbardef

$ ninja -C build/
ninja: Entering directory `build/'
[1/1] cd /home/fabian/Source/woof/base && /usr/bin/cmake -E tar cf /home/fabian/Source/woof/build/src/woof.pk3 --format=zip .

$ ninja -C build/
ninja: Entering directory `build/'
[1/1] cd /home/fabian/Source/woof/base && /usr/bin/cmake -E tar cf /home/fabian/Source/woof/build/src/woof.pk3 --format=zip .
```

Reply:
Fixed in https://github.com/fabiangreffrath/woof/pull/1916/commits/0f1f8d921720b2e48eb63e7a6269eac0adb91921

Reply:
Thank you!